### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rock3r**
Update AGP 4.1 to alpha 04, Gradle to 6.3-rc1, and Kotlin to 1.3.71. App smoke tested and working.

CC @objcode

### CODE REVIEW

File: JetNews/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Thu Feb 27 20:17:30 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-rc-1-all.zip
```
Comment (nickbutcher) [inline JetNews/gradle/wrapper/gradle-wrapper.properties]: 6.3 is out (i.e. no need for RC) and we tend to use `bin` releases.

Comment (rock3r) [reply JetNews/gradle/wrapper/gradle-wrapper.properties]: Ah, didn't know it was out. Just went with what AS told me to in the quickfix. Will update and switch to bin. Out of curiosity, any reason why you use `bin`? AFAIK `all` is preferred for usage in IDEs, and AS itself suggests to switch to `all`.

Comment (nickbutcher) [reply JetNews/gradle/wrapper/gradle-wrapper.properties]: IIRC it's just because it's a smaller download if you don't need source.

**Comment by nickbutcher** [comment]
Ok, now this one **does** need a rebase.

**Comment by rock3r** [comment]
Done :)

**Pull request merged**